### ISSUE TITLE
Add sanity check that contract can deploy on Ropsten

### ISSUE THREAD

**Issue by boolafish** [omgnetwork/plasma-contracts#632]
##  Issue Type 
```
[ ] bug report
[x] feature request
```

## Current Behavior

From Devops' feedback, contract deployment via truffle migrate from time to time fails and requires manual retry. Would be nice to have automated sanity check testing the contract deployment on real network.

PS. currently it is only tested against local ganache, which the response time is fast and does not need to wait on the transaction to go through the network + mining....etc

## Motivation

Make devops happy

**Comment by boolafish** [comment]
done, have the deployment to Rinkeby test
